pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -151,18 +194,23 @@ func getMonitoringConsoleStatefulSet(client splcommon.ControllerClient, cr splco
 
 	env := []corev1.EnvVar{}
 
+	if MultiSite == "true" {
+		env = append(env, corev1.EnvVar{Name: "SPLUNK_SITE", Value: "site0"}, corev1.EnvVar{Name: "SPLUNK_MULTISITE_MASTER", Value: GetSplunkServiceName(SplunkClusterMaster, cr.GetName(), false)})
+	}
+
 	// append labels and annotations from parent
 	splcommon.AppendParentMeta(statefulSet.Spec.Template.GetObjectMeta(), cr.GetObjectMeta())
 
 	// update statefulset's pod template with common splunk pod config
 	updateSplunkPodTemplateWithConfig(client, &statefulSet.Spec.Template, cr, spec, instanceType, env, secretName)
 
 	//update podTemplate annotation with configMap resource version
-	var monitoringConsoleConfigMap *corev1.ConfigMap
+	//use emptyEnv here to avoid adding site amd multisite_master info in configMap
+	//TODO: Update this configMap retrieval by API from PR #134
 	if cr.GetObjectMeta().GetDeletionTimestamp() != nil {
-		monitoringConsoleConfigMap, _ = ApplyMonitoringConsoleEnvConfigMap(client, cr.GetNamespace(), cr.GetName(), env, false)
+		monitoringConsoleConfigMap, _ = ApplyMonitoringConsoleEnvConfigMap(client, cr.GetNamespace(), cr.GetName(), emptyEnv, false)
 	} else {
-		monitoringConsoleConfigMap, _ = ApplyMonitoringConsoleEnvConfigMap(client, cr.GetNamespace(), cr.GetName(), env, true)
+		monitoringConsoleConfigMap, _ = ApplyMonitoringConsoleEnvConfigMap(client, cr.GetNamespace(), cr.GetName(), emptyEnv, true)
```

Comment:
so if we are passing `emptyEnv` in this function, then how will we provide `newURLs` to the configMap?